src/games/index.js

```diff
@@ -1,9 +1,25 @@
-const Test1 = require('./test-1');
-const Test2 = require('./test-2');
 const Draw = require('./draw');
+const MoveTest = require('./move-test');
```

Comment:
I think we should change this to `export const ....`
otherwise we are going to have a lot of duplicated names in this file 
see: https://stackoverflow.com/questions/42461330/difference-between-export-const-foo-export-default-foo-and-module-exports-foo

Reply:
Can you give an example of what that would look like? Also I don't know what you mean by duplicated names, wouldn't we always have the problem of needing unique game class names?

Reply:
my bad, I mistook something in es-6 for something native to node. your approach is correct my bad!